codalab/lib/cli_util.py

```diff
@@ -1,6 +1,33 @@
 import re
 
-from codalab.common import UsageError
+from codalab.common import precondition, UsageError
+
+
+def safe_get(o, *args, **kwargs):
```

Comment:
given that this lives in `cli_util.py`, the name of this function should probably be a bit more specific (contain nested_dict or something)